### PR TITLE
[BOG-2021] Event creation devopsdays bogota 2021

### PR DESCRIPTION
**Pull request by rrmartinez87**
Hello;

We are created the 2021 devopsdays Bogota

### CODE REVIEW

File: data/events/2021-bogota.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  - name: contact
+  - name: conduct
+# - name: example
+#   icon: "map-o" # This is a font-awesome icon that will display on small screens. Choose at http://fontawesome.io/icons/
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
+  - name: "John Doe"
```
Comment (MikeRosTX) [inline data/events/2021-bogota.yml:52]: Add the organizers.

File: data/events/2021-bogota.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+#
+location_address: "" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+ # - name: propose
+ # - name: location
+ # - name: registration
+ # - name: program
+ # - name: speakers
+  - name: sponsor
```
Comment (MikeRosTX) [inline data/events/2021-bogota.yml:42]: Remark # out sponsor until you are ready with the prices.

**Review by MikeRosTX — CHANGES_REQUESTED** [review]

File: data/events/2021-bogota.yml (hunk 1 above)
Comment (rrmartinez87) [reply data/events/2021-bogota.yml]: Organizers Info added
Thanks

**Review by MikeRosTX — APPROVED** [review]

File: data/events/2021-bogota.yml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+ # - name: sponsor
+  - name: contact
+  - name: conduct
+# - name: example
+#   icon: "map-o" # This is a font-awesome icon that will display on small screens. Choose at http://fontawesome.io/icons/
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
```
Comment (MikeRosTX) [inline data/events/2021-bogota.yml:51]: All organizers pictures are missing.

**Review by MikeRosTX — CHANGES_REQUESTED** [review]

**Review by MikeRosTX — APPROVED** [review]

**Pull request merged**